### PR TITLE
More membership features

### PR DESCRIPTION
**Pull request by mayeaux**
- Allows user to select currency on payment method page that is persisted in shared client settings
- Back to membership page button
- Improves fiat support tipping functionality

### CODE REVIEW

File: ui/component/walletTipAmountSelector/view.jsx [hunk 1]
```diff
@@ -304,16 +306,19 @@ function WalletTipAmountSelector(props: Props) {
...
               <>
                 <Button navigate={`/$/${PAGES.SETTINGS_STRIPE_CARD}`} label={__('Add a Card')} button="link" />
                 {' ' + __('To Tip Creators')}
-              </>
+              </>,
+              'add-a-card-help-message'
             )
           : !canReceiveFiatTip
-          ? getHelpMessage(__('Only creators that verify cash accounts can receive tips'))
+          ? getHelpMessage(__('Only creators that have added a bank account can receive tips'))
```
Comment (mayeaux) [inline ui/component/walletTipAmountSelector/view.jsx:321]: I'm suggesting to change the copy here, it's unclear to me what a 'cash account' is and the tense is wrong as well

File: ui/scss/component/_wallet-tip-send.scss [hunk 2]
```diff
@@ -2,3 +2,19 @@
   margin-top: -21px;
   margin-bottom: -21px;
 }
+
+.wallet-send-tip-modal {
+  .card__title {
+    margin-top: -16px;
```
Comment (mayeaux) [inline ui/scss/component/_wallet-tip-send.scss:8]: If you'd like @infinite-persistence I can take a look at adding these using `_vars` `--spacing` variables

File: ui/component/walletSendTip/view.jsx [hunk 3]
```diff
@@ -78,8 +78,8 @@ export default function WalletSendTip(props: Props) {
   } = props;
 
   /** WHAT TAB TO SHOW **/
-  // set default tab to for new users based on if it's their claim or not
-  const defaultTabToShow = claimIsMine ? TAB_BOOST : TAB_LBC;
+  // if it's your content, we show boost, otherwise default is LBC
+  const defaultTabToShow = claimIsMine ? TAB_BOOST : TAB_FIAT;
```
Comment (tzarebczan) [inline ui/component/walletSendTip/view.jsx:82]: This should probably only default to fiat if the creator has fiat enabled.

File: ui/component/walletTipAmountSelector/view.jsx (hunk 1 above)
Comment (tzarebczan) [reply ui/component/walletTipAmountSelector/view.jsx:321]: Any time we do this, we have to be aware of translations, and having everyone retranslate. I'd only want to do this when absolutely necessary...I think it's fine as is.

**Pull request merged**